playbooks/robusta_playbooks/common_actions.py

```diff
@@ -53,7 +54,7 @@ def customise_finding(event: ExecutionBaseEvent, params: FindingOverrides):
 
     aggregation_key = Template(params.aggregation_key).safe_substitute(labels) if params.aggregation_key else None
 
-    event.override_finding_attributes(title, description, severity, aggregation_key)
+    event.override_finding_attributes(title, description, severity, to_pascal_case(aggregation_key))
```

Comment:
I think here we don't want to change it, because this is something the user is providing